### PR TITLE
[PLAY-943] Website V1 - Site Footer 

### PR DESCRIPTION
**Pull request by nickamantia**
[PLAY-943](https://nitro.powerhrg.com/runway/backlog_items/PLAY-943)

This PR:
Implements new footer design
Adds `textAlign` global prop
Adds `color` prop to the `hover` prop

Desktop:
Screenshot 2023-09-05 at 3 46 25 PM

Tablet:
Screenshot 2023-09-05 at 3 46 40 PM

Mobile:
Screenshot 2023-09-05 at 3 46 54 PM

**How to test?** Steps to confirm the desired behavior:
1. Go to any page and scroll down to the footer.
2. add `textAlign` or `text_align` with a value and see if you get the results you were expecting.

#### Checklist:
- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook-website/app/javascript/site_styles/_site-style.scss [hunk 1]
```diff
@@ -164,27 +164,26 @@ body {
...
+    background-color: $bg_dark;
+    padding: $space_lg * 2 0;
     margin-top: auto;
     .inner-footer-wrapper {
-      max-width: 1350px;
       margin: 0 auto;
     }
+    .link-container {
+      width: 100%;
+    }
```
Comment (nickamantia) [inline playbook-website/app/javascript/site_styles/_site-style.scss:175]: Is this fine to have until we create the width prop?

Comment (jasperfurniss) [reply playbook-website/app/javascript/site_styles/_site-style.scss:175]: Yeah. I think it's fine for now, and we will adjust it once I get the `width` global prop sorted out.

File: playbook-website/app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -1,49 +1,68 @@
...
+          <a href="https://nitro.powerhrg.com/connect#rooms/23514">
+            <%= pb_rails("body", props: { text: "Connect Chat", dark: true, color: "light", margin_bottom: "md", text_hover: true })  %>
+          </a>
         <% end %>
       <% end %>
     <% end %>
+
     <div class="copyright-tag">
-      <%= pb_rails("body", props: { text: "© Power Home Remodeling", color: "lighter" }) %>
+      <%= pb_rails("caption", props: { text: "© 2023-24, ALL RIGHTS RESERVED", color: "lighter" }) %>
```
Comment (jasoncypret) [inline playbook-website/app/views/layouts/_footer.html.erb:65]:
```suggestion
      <%= pb_rails("caption", props: { text: "© 2023-24, ALL RIGHTS RESERVED", color: "lighter", dark: true }) %>
```

File: playbook-website/app/javascript/site_styles/_site-style.scss [hunk 3]
```diff
@@ -164,27 +164,27 @@ body {
   }
 
   &--footer {
-    background-color: $royal;
-    padding: $space_lg * 2 $space_lg * 3;
+    background-color: $bg_dark;
+    padding: $space_lg * 2 0 40px;
```
Comment (jasoncypret) [inline playbook-website/app/javascript/site_styles/_site-style.scss:168]: Background kit could have replaced this.

File: playbook/app/pb_kits/playbook/pb_body/_body.scss [hunk 4]
```diff
@@ -42,6 +42,10 @@
     @include body_truncate;
   }
 
+  &[class*=_text_hover]:hover {
+    color: $white !important;
+  }
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_body/_body.scss:47]: I'm not sure I understand this addition, and the need for the important tag. Is this needed?

File: playbook/app/pb_kits/playbook/pb_title/_title.scss [hunk 5]
```diff
@@ -56,4 +52,7 @@
         }
     }
   }
+  &.dark {
+    @include pb_title_dark;
+  }
```
Comment (nickamantia) [inline playbook/app/pb_kits/playbook/pb_title/_title.scss:57]: fix for this title responsive prop bug i found: [video](https://share.zight.com/Blu2jvqA)

File: playbook/app/pb_kits/playbook/pb_table/docs/_table_two_actions.jsx [hunk 6]
```diff
@@ -28,7 +28,7 @@ const TableOneAction = (props) => {
             <Button
                 onClick={() => alert('button clicked!')}
                 paddingLeft="none"
-                text="Tetriary Action"
+                text="Tertiary Action"
```
Comment (nickamantia) [inline playbook/app/pb_kits/playbook/pb_table/docs/_table_two_actions.jsx:31]: Fixing typos

File: playbook/app/pb_kits/playbook/pb_body/_body.scss (hunk 4 above)
Comment (nickamantia) [reply playbook/app/pb_kits/playbook/pb_body/_body.scss]: Talked to Jasper and we decided to move this to the `hover` global prop.

**Review by nidaqg — APPROVED** [review]
🚀

File: playbook/app/pb_kits/playbook/utilities/globalProps.ts [hunk 7]
```diff
@@ -62,6 +62,7 @@ type FlexWrap = {
 
 type Hover = Shadow & {
   background?: string,
+  color?: string,
```
Comment (nidaqg) [inline playbook/app/pb_kits/playbook/utilities/globalProps.ts:65]: Should we add this addition to the Hover global prop to the Hover docs on the visual guidelines page? That can definitely be a follow-up story though, doesn't have to be on this PR

cc: @jasperfurniss

**Review by jasperfurniss — DISMISSED** [review]

**Review by nidaqg — APPROVED** [review]

**Pull request merged**